develop/schemadoc/Protocol/Protocol.Params.Param.Interprete.RawType.md

```diff
@@ -111,26 +99,19 @@ Text consisting of the following characters:
 - !
 - a space (“ “)
 
-Example:
-
-
-```xml
-<RawType>text</RawType>
-```
+> [!NOTE]
+> text is depricated. It can still be used but is not recommended for new developments. We advice to use other instead.
 
 ### unsigned number
 
-DataMiner will convert the hex value to a number using a decimal conversion.
+DataMiner will convert the value to a number using a decimal conversion.
 
-Example: If the hex value is 31, the decimal conversion gives 49. The parameter will contain 49 (result will be the same as if you use your calculator to convert from hex to dec).
+Example: If the binary value is 0x31, the decimal conversion gives 49. The parameter will contain 49 (result will be the same as if you use your calculator to convert from hex to dec).
 
+todo sentence below is confusing and incorrect when using different lengths
 Numbers between 0 and 255.
```

Comment:
Add list of options?
Length 1: 0 - 255
Length 2: 0 - 65535
Length 4: 0 - 4294967295
Do we support length 8?